scripts/run-functional-tests

```diff
@@ -42,10 +42,16 @@ install_packages() {
 
 run_tests() {
     log_progress "Running tests"
-    pytest --disable-pytest-warnings -vvv tests/functional
+    pytest --disable-pytest-warnings -vvv tests/functional "${@}"
 }
 
+isolate() {
+    export GIT_CONFIG_GLOBAL=''
+    export GIT_CONFIG_SYSTEM=''
```

Comment:
I just found out we already hit that issue once in the past, but it was only fixed for the `unittest` Makefile target. I think we should unify this and add an autouse fixture to unset the variables in `tests/conftest.py` instead.